scss/bitstyles/utilities/flex/flex.stories.mdx

```diff
@@ -147,26 +276,46 @@ And this time the same layouts, with images as content (any element with intrins
 
 ## Flex wrap
 
-Give the flex parent `.u-flex--wrap` too, if you want the children to wrap onto a new line when they run of space. This can be a good thing to do when you have a list of elements with intrinsic or fixed size e.g. a list of buttons, icons, or images. The result will mean the elements are allowed to flow onto the next line when there’s not enough space for them to be adjacent.
+Give the flex parent `.u-flex-wrap`if you want the children to wrap onto a new line when they run of space. This can be a good thing to do when you have a list of elements with intrinsic or fixed size e.g. a list of buttons, icons, or images. The elements will flow onto the next line when there’s not enough space for them to be adjacent.
```

Comment:
```suggestion
Give the flex parent `.u-flex-wrap`, if you want the children to wrap onto a new line when they run of space. This can be a good thing to do when you have a list of elements with intrinsic or fixed size e.g. a list of buttons, icons, or images. The elements will flow onto the next line when there’s not enough space for them to be adjacent.
```